package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "ls-next-app",
   "version": "0.1.0",
+  "liscense": ISC,
```

```suggestion
  "license": "MIT",
```

Added the supporting LICENSE.md, which I put as MIT. Let's update that here.